BUILD

```diff
@@ -0,0 +1,57 @@
+load(
+    "@org_tensorflow//tensorflow:tensorflow.bzl",
+    "tf_cc_shared_object",
+)
+
+tf_cc_shared_object(
+    name = "_XLAC.so",
+    copts = [
+        "-DTORCH_API_INCLUDE_EXTENSION_H",
+        "-DTORCH_EXTENSION_NAME=_XLAC",
+        "-fopenmp",
+        "-fPIC",
+        "-fwrapv",
+    ],
+    linkopts = [
+        "-Wl,-rpath,$$ORIGIN/torch_xla/lib",  # for libtpu
+    ],
+    visibility = ["//visibility:public"],
+    deps = [
+        "//third_party/xla_client:computation_client",
+        "//third_party/xla_client:mesh_service",
+        "//third_party/xla_client:metrics",
+        "//third_party/xla_client:metrics_analysis",
+        "//third_party/xla_client:metrics_reader",
+        "//third_party/xla_client:multi_wait",
+        "//third_party/xla_client:profiler",
+        "//third_party/xla_client:record_reader",
+        "//third_party/xla_client:sys_util",
+        "//third_party/xla_client:thread_pool",
+        "//third_party/xla_client:util",
+        "//third_party/xla_client:xla_util",
+        "//torch_xla/csrc:computation",
+        "//torch_xla/csrc:device",
+        "//torch_xla/csrc:init_python_bindings",
+        "//torch_xla/csrc:tensor",
+        "//torch_xla/csrc:version",
+        "@com_google_absl//absl/container:flat_hash_map",
+        "@com_google_absl//absl/strings",
+        "@com_google_absl//absl/types:variant",
+        "@org_tensorflow//tensorflow/compiler/xla/python/profiler/internal:traceme_wrapper",
```

Comment:
are this `"@org_tensorflow//tensorflow/compiler/xla/python/profiler/internal:traceme_wrapper"` is different from the following `"@org_tensorflow//tensorflow/python/profiler/internal:profiler_pywrap_impl",`?

do we have the difference in their usage?

Reply:
I don't really know, I just copied whatever was used before.

Reply:
`@org_tensorflow//tensorflow/python/profiler/internal:profiler_pywrap_impl` sounds like the python bindings that JAX would use, but I don't know for sure. Let's keep this PR as minimal as possible (ie still huge) and change as few dependencies as possible.